src/Shaders/ShadersInclude/pbrBlockFinalLitComponents.fx

```diff
@@ -34,7 +34,7 @@
     #endif
 
     #if defined(SS_TRANSLUCENCY)
-        finalIrradiance *= (1.0 - subSurfaceOut.transmittance);
+        finalIrradiance *= (1.0 - vSubSurfaceIntensity.y);

```

Comment:
The previous code results in all non-transmitted light to be reflected diffusely but this isn't really what we want, I don't think. If translucency is fully on, all non-transmitted light should be absorbed, not reflected. Therefore, I think the right thing to do here is multiply by `1.0 - translucencyIntensity`

Reply:
Don't know for this one, will let @sebavan answer.

However, I would think that `translucency is fully on` means the material is fully transparent (https://en.wikipedia.org/wiki/Transparency_and_translucency), so transmittance=1 and then there's no light reflected diffusely => `1.0 - subSurfaceOut.transmittance` would be the right factor in that case.

Reply:
Not sure as well,

but like @Popov72  1 - transmittance sounds like better here. I agree that some part should be absorbed though but not sure what amount. I do not think that we should anyway use intensity here cause it would prevent any parts like ears or nostrils where both are heavily at play.

Reply:
Maybe we need a special value like transmittance e.g. absorbance and multiply as well by 1 - absorbance in case of SSS ???

Reply:
The issue is that `transmittance` takes absorption into account so `1.0 - transmittance` is the amount of light not absorbed. This means that that light is reflected instead and a 100% translucent material reflects a lot of light.
`translucencyIntensity` is the percentage of light that passes into the surface so I think it's the right value to use here.

Here is an example of a 100% translucent material with a red tintColor:
With `1.0 - transmittance`:
<img width="704" alt="Screen Shot 2020-11-10 at 11 55 08 AM" src="https://user-images.githubusercontent.com/1069125/98726812-d780e480-234b-11eb-9e23-754d97df2ac5.png">

With `1.0 - translucentIntensity`:
<img width="764" alt="Screen Shot 2020-11-10 at 11 53 18 AM" src="https://user-images.githubusercontent.com/1069125/98726843-e4053d00-234b-11eb-9eff-7a8f9904a125.png">

The second one is what I would expect.

Reply:
Could you share how it would look on the head you used previously ? just for comparison purpose ?

Reply:
I actually think you might be right @MiiBond Let s see on the head what it loooks and use this into the release if it looks ok ?

Reply:
@Popov72 , @MiiBond we have only tomorrow left for the change we put in 4.2 would be great to decide which one to put in to prevent adding an extra back compat flag ? I propose @MiiBond you check on the previous head but my guess is it will be similar to before ? actually, at least it should preserve energy :-)

And then we can push ?

Reply:
I will let you judge as I don't know what a 100% translucency material should do (on my mind it should be fully transparent but obviously it is not in the screenshots) :)

Reply:
Ok after a few test the translucency intensity looks better, tried on a pure sphere and the other method looks a bit weird. We ll revisit if needed in 5.0, but I ll make a PR for 4.2 doing what Mike proposed here but reusing our computed intensity from the maps. PR should be here shortly.

Reply:
https://github.com/BabylonJS/Babylon.js/pull/9390

Reply:
Awesome! Thanks.